### PR TITLE
fix(DEVELOPERS): grammar

### PR DESCRIPTION
**Pull request by gargarchit**
Signed-off-by: Archit garg4.ag@gmail.com

I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.
Please Merge this

### CODE REVIEW

**Comment by gargarchit** [comment]
Thank You @irmerk I have update changes and Add DCO

File: DEVELOPERS.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ We have very precise rules over how our git commit messages can be formatted.  T
 readable messages** that are easy to follow when looking through the **project history** and **git logs**. 
 But also, we use the git commit messages to **generate the Ergo change log**.
 
-The commit message formatting can be added using a version of typical git workflow.
+The commit message formatting can be added using a version of the typical git workflow.
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

**Comment by irmerk** [comment]
One change, then this can be merged.

File: DEVELOPERS.md (hunk 1 above)
Comment (gargarchit) [reply DEVELOPERS.md]: Done

**Comment by gargarchit** [comment]
@irmerk I have updated changes according to requirements

**Comment by irmerk** [comment]
Now this needs DCO signoff

**Comment by gargarchit** [comment]
@irmerk Done I have Added DCO

**Comment by gargarchit** [comment]
@irmerk I have Updated all changes

**Comment by gargarchit** [comment]
Thank You @irmerk

**Pull request merged**